Dockerfile

```diff
@@ -46,13 +47,15 @@ RUN \
   rm -Rf /app/bazarr/bin/bin && \
   echo "UpdateMethod=docker\nBranch=development\nPackageVersion=${VERSION}\nPackageAuthor=[linuxserver.io](https://linuxserver.io)" > /app/bazarr/package_info && \
   echo "**** Install requirements ****" && \
-  pip3 install --no-cache-dir -U  -r \
+  pip3 install -U --no-cache-dir pip && \
+  pip3 install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/alpine/  -r \
```

Comment:
This should be just `pip` otherwise it refers to the system version, which is behind